platforms/hyperledger-indy/scripts/indy-ledger-script/vault-setup.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
This script installs the Vault itself, why do we need such a script? I hope you don't plan to install Vault into the Docker container running from the indy-cli Docker image...

Reply:
vault login will not work inside container if vault is not setup.then how will the vault commands run inside docker cli?

Reply:
If there isn't a standalone Vault CLI which doesn't involve the installation of the whole Vault then use the REST API to access the Vault.